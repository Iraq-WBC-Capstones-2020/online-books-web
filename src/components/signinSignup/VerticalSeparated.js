import React from 'react';
export default function VerticalSeparated() {
  return (
    <div class="mt-5 flex flex-col content-center h-auto">
      <div class="text-center">
        <div className="w-0 verline inline-block"></div>
      </div>
      <div className="h-20 text-center">
        <span className="h-20 w-20 bg-gray-100 inline-block rounded-full orr">
          or
        </span>
      </div>
      <div class="text-center">
        <div className="w-0 verline inline-block"></div>
      </div>
    </div>
  );
}
