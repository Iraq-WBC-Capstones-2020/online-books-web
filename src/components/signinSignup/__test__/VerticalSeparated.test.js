import React from 'react';
import renderer from 'react-test-renderer';
import VerticalSeparated from './../VerticalSeparated';

test('VerticalSeparated snapshot', () => {
  const tree = renderer.create(<VerticalSeparated />).toJSON();
  expect(tree).toMatchSnapshot();
});
