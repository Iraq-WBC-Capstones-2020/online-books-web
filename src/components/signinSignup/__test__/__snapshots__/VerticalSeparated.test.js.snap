// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerticalSeparated snapshot 1`] = `
Array [
  <div
    className="container"
    data-test="container"
  >
    <div data-test="col"  className="col-md-5">
      <div className="verline d-none d-sm-block"></div>
      <div className=" orr"  data-test="card">
        <div  className="card-body" data-test="card">
          <div className="text-center">
            <div className="rounded-pill px-2"> or </div>
          </div>
        </div>
      </div>
      <div className="verline2 d-none d-sm-block"></div>
    </div>
   
]
`;
